src/main/java/FtpServer.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Created by webonise on 13/9/16.
+ */
+public class FtpServer implements Server {
+
+    boolean isRunning = false;
+
+    @Override
+    public boolean isServerRunning() {
+        return isRunning;
+    }
+
+    @Override
+    public void startServerAndListenForRequests() {
```

Comment:
Thanks ameya. I will take care not to check empty methods again. This method is for starting the the ftp server and list of incoming socket connections. You can refer same method in HttpServer to have an idea of what I actually wanted to do. In this case I didn't have any idea how to start a ftp server. So I left this method as part of design. Will not repeat again. (Y)